code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -283,33 +283,33 @@
 	var/mob/living/carbon/victim = M
 	if(method == TOUCH || method == VAPOR)
 		//check for protection
-		var/mouth_covered = victim.is_mouth_covered()
 		var/eyes_covered = victim.is_eyes_covered()
 
 		//actually handle the pepperspray effects
-		if ( eyes_covered && mouth_covered )
+		if (eyes_covered && istype(victim.wear_mask, /obj/item/clothing/mask/gas))
```

Comment:
This actually results gas masks being better than sunglasses against pepperspray. If I read the description correctly, that was not the intent?

Reply:
Yeah I dont know, I tried to balance the effects, but gas masks are the only mouth protection with eye protection against pepper spray.